File: week-02/day-01/draw_triangle.py
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was

number1 = int(input("First number: "))
star = "*"
for j in range(1, (number1+1)):
    print(star*(j))

File: week-08/day-01/28_draw_triangle/draw_triangle.js
var lineCount = 4;
var char = "*";

// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is

for(i=0;i<lineCount;i+=1){
    console.log(char.repeat(i+1));
}